v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -89,6 +89,8 @@ spec:
          - exposedPort: 13132
          - exposedPort: 13133
      memoryLimit: "512M"
+     cpuLimit: "1000m"
```

Comment:
As commented elsewhere....we don't want to limit theia to 1 CPU either.

Reply:
are you absolutely sure that theia is designed to be multicore supported?

Reply:
3%, even worse....no useful work will happen with 3% of a cpu.

Reply:
completely agree with @tsmaeder that 3% is too small

Reply:
@tsmaeder @vitaliy-guliy do you folks understand the difference between a request & limit?

Reply:
Yes, but if the scheduler sees a 3% request, it might schedule > 30 of those containers for a single cpu, I doubt that we want that.

Reply:
@tsmaeder we are currently failing to schedule even one unique pod and you worry that we "might" schedule 30? Schedulers consider cpu limits as well and won't let you schedule pods if the max cpu limit is exceeded (as you have experienced on crc).

Reply:
That may be, but it's has nothing to do with the question what the right cpuRequest value is.. I still don't want to expose our users to a theia that runs on 3% of a single core.

Reply:
It won't run on 3% of a single core. 3% will be reserved but the container will be able to use up to the CPU limit. What's the right number for you? Consider that currently the default CPU request for che-theia is 0%. We are exposing our users to a theia that reserves 0% of CPU.